validator/remote/src/test/java/tech/pegasys/teku/validator/remote/apiclient/OkHttpValidatorRestApiClientTest.java

```diff
@@ -232,6 +237,32 @@ public void createUnsignedBlock_WhenSuccess_ReturnsBeaconBlock() {
     assertThat(beaconBlock.get()).usingRecursiveComparison().isEqualTo(expectedBeaconBlock);
   }
 
+  @Test
+  public void createUnsignedBlocks_shouldUseBlindedBlocksIfEnabled() {
+    final Spec spec = TestSpecFactory.createMinimalBellatrix();
+    final DataStructureUtil dataStructureUtil = new DataStructureUtil(spec);
+    final UInt64 slot = UInt64.ONE;
+    final BLSSignature blsSignature = schemaObjects.blsSignature();
+    final Optional<Bytes32> graffiti = Optional.of(Bytes32.random());
+    final tech.pegasys.teku.spec.datastructures.blocks.BeaconBlock expectedBeaconBlock =
+        dataStructureUtil.randomBlindedBeaconBlock(slot);
+
+    mockWebServer.enqueue(
+        new MockResponse()
+            .setResponseCode(SC_OK)
+            .setBody(
+                asJson(
+                    new GetNewBlindedBlockResponse(
+                        SpecMilestone.BELLATRIX, new BlindedBlockBellatrix(expectedBeaconBlock)))));
+    Optional<BeaconBlock> beaconBlock =
+        apiClient.createUnsignedBlock(slot, blsSignature, graffiti, true);
+    assertThat(beaconBlock).isPresent();
+
+    // TODO make comparable
+    // Some of the underlying stores are mutable, but the values are the same.
+    // It does make it harder to compare here though
```

Comment:
Might need to do something with this TODO?

Reply:
its compared above so I'm not super worried. when it not working it goes boom... it'd be nice to do...